tests/e2e-openshift/otlp-metrics-traces/verify-traces.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: verify-traces
+spec:
+  template:
+    spec:
+      containers:
+      - name: verify-traces
+        image: ghcr.io/grafana/tempo-operator/test-utils:main
+        command: ["/bin/bash", "-eux", "-c"]
+        args:
+        - |
+          curl -sS -G \
+            --data-urlencode 'q={ resource.service.name="telemetrygen" }' \
+            http://tempo-otlpmetrics.chainsaw-otlp-metrics.svc:3200/api/search \
+            | tee /tmp/tempo.out
+
+          num_traces=$(jq ".traces | length" /tmp/tempo.out)
+          if [[ "$num_traces" -le "1" ]]; then
+            echo && echo "The Tempo API returned $num_traces."
+            exit 1
+          fi
+      restartPolicy: Never
```

Comment:
Can this be a problem in those situations where the infra is flaky and you want some retries while chainsaw is running?

Reply:
The pods will be recreated when it fails, by the job until the default backofflimit is reached.